pa_config/rca_idle_master.conf

```diff
@@ -109,22 +109,49 @@
   // Default Values are added in the config specific file as well
   // Modify them as well whenever changing the default values (DeciderConfig.java)
   "decider-config-settings": {
-    // Decreasing order of priority for the type of workload we can expect on the cluster.
-    // Priority order in the list goes from most expected to the lease expected workload type.
     "workload-type": {
-      "priority-order": ["ingest", "search"]
+      "prefer-ingest": true,
+      "prefer-search": true
     },
     // Decreasing order of priority for the type of cache which is expected to be consumed more.
     // Priority order in the list goes from most used to the lease used cache type.
     "cache-type": {
       "priority-order": ["fielddata-cache", "shard-request-cache", "query-cache", "bitset-filter-cache"]
     },
-    // cache decider - Needs to be updated as per the performance test results
-    "cache-bounds": {
-      "field-data-cache-upper-bound" : 0.4,
-      "shard-request-cache-upper-bound" : 0.05
+    "old-gen-decision-policy-config": {
+      "queue-bucket-size" : 20,
+      "old-gen-threshold-level-one" : 0.6,
+      "old-gen-threshold-level-two" : 0.75,
+      "old-gen-threshold-level-three" : 0.9
     }
   },
+  // Action Configurations
```

Comment:
Do we need to add all of these configurations? Wont just changing the default queue lower bounds suffice?

Reply:
We don't have to add all configurations in the rca conf but I remember Joydeep once asked me to explicitly add all rca conf into the file so that people who are not familiar with the settings can easily understand and set it